peps/pep-0719.rst

```diff
@@ -51,6 +51,9 @@ Actual:
 - 3.13.0 candidate 2: Friday, 2024-09-06
 - 3.13.0 candidate 3: Tuesday, 2024-10-01
 - 3.13.0 final: Monday, 2024-10-07
+
+Expected:
+
 - 3.13.1: Tuesday, 2024-12-03
 - 3.13.2: Tuesday, 2025-02-04
 - 3.13.3: Tuesday, 2025-04-08
```

Comment:
Can you move this section under the "Bugfix releases" header?

So we have:

```rst
3.13.0 schedule
---------------

Actual:

[only 3.13.0 stuff]

Bugfix releases
---------------

Actual:

[3.13.x in the past]

Expected:

[3.13.x in the future]
```

Reply:
Updated in 791e0e95e3a6539753bbc4238f081580e097de56